saleor/graphql/schema.graphql

```diff
@@ -3256,6 +3256,7 @@ enum AttributeChoicesSortField {
 
 input AttributeValueFilterInput {
   search: String
+  ids: [ID]
```

Comment:
If we merged [this](https://github.com/saleor/saleor/pull/9391) first and then rebased this PR, `ids` field would automatically become `[ID!]` which is the desired type here.

Reply:
This change should be done in `GlobalIDMultipleChoiceFilter` class.